### PR TITLE
BI-1413-Remove-company-specific-info

### PR DESCRIPTION
**Pull request by nguyentranlogigear**
Currently, some tickets about rspec that uses same data with company info. So the purpose of ticket is remove or changed it

### CODE REVIEW

File: workspace/spec/lita/handlers/teamcity_spec_anki.rb [hunk 1]
```diff
@@ -2,35 +2,31 @@
 
 describe Lita::Handlers::Teamcity, lita_handler: true do
   before do
-    registry.config.handlers.teamcity.site             = 'https://build.ankicore.com'
+    registry.config.handlers.teamcity.site             = 'https://site.teamcity.com'
     registry.config.handlers.teamcity.username         = 'username'
     registry.config.handlers.teamcity.password         = 'password'
-    registry.config.handlers.teamcity.git_uri          = 'git@github.com:anki'
+    registry.config.handlers.teamcity.git_uri          = 'git@github.com:git_url'
```
Comment (nguyentunglam912) [inline workspace/spec/lita/handlers/teamcity_spec_anki.rb]: Should be : 'git@github.com:git_uri' to match with registry.config.handlers.teamcity.git_uri

File: workspace/spec/lita/handlers/teamcity_spec_anki.rb [hunk 2]
```diff
@@ -157,8 +155,8 @@
   end
 
   describe '#artifacts_by_build_id' do
```
Comment (nguyentunglam912) [inline workspace/spec/lita/handlers/teamcity_spec_anki.rb]: Should updating space of variables for all 'it' of describe '#artifacts_by_build_id'

Comment (nguyentranlogigear) [reply workspace/spec/lita/handlers/teamcity_spec_anki.rb]: I updated it

**Comment by nguyentunglam912** [comment]
LGTM

**Review by scopp — APPROVED** [review]
LGTM

**Pull request merged**